dspace-server-webapp/src/main/java/org/dspace/app/rest/security/CustomLogoutHandler.java

```diff
@@ -45,7 +45,7 @@ public void logout(HttpServletRequest httpServletRequest, HttpServletResponse ht
                        Authentication authentication) {
         try {
             Context context = ContextUtil.obtainContext(httpServletRequest);
-            restAuthenticationService.invalidateAuthenticationData(httpServletRequest, context);
+            restAuthenticationService.invalidateAuthenticationData(httpServletRequest, httpServletResponse, context);
```

Comment:
A verification can be used to identify if the current session is a shibboleth session and then call the configured logout parameter